CONTRIBUTORS.md

```diff
@@ -0,0 +1,10 @@
+
+2006-2010	xcherif 	- Alexandre Cherif
+2006-2010   Max
+Mar 2016	ynamiki		
+Mar 2016	vlsi		- Vladimir Sitnikov
+Apr	2016	scr34m		- Győrvári Gábor
+Apr	2016	pitterling	- Peter Pitterling
+Nov	2016	vest
+Mar	2017	ikedaj		- Junko IKEDA
+May 2017    elkrieg
```

Comment:
The above does not look like a markdown format.
At least GitHub renders it in a weird way: https://github.com/Pitterling/ksar/blob/f39cc47db31e0123a0fc2cd5ba1603ca9ddf80d1/CONTRIBUTORS.md